app/build.gradle

```diff
@@ -99,10 +99,8 @@ dependencies {
     implementation "com.squareup.okhttp3:logging-interceptor:${rootConfiguration.okHttpVersion}"
 
     //For Link Previews
-    implementation "org.jsoup:jsoup:${rootConfiguration.jsoupVersion}"
-
-    // required
-    implementation "com.leocardz:link-preview:${rootConfiguration.linkPreviewVersion}"
+    implementation "org.jsoup:jsoup:${rootConfiguration.jsoupVersion}" // required
```

Comment:
Remove

Reply:
@iamareebjamal the library requires the use of jsoup, Initially I removed it but then I saw in the code of the library that there was the usage of jsoup so it was needed.

Reply:
Alright